drivers/SmartThings/samsung-audio/profiles/samsung-audio.yml

```diff
@@ -4,6 +4,18 @@ components:
   capabilities:
   - id: mediaPlayback
     version: 1
+    config:
+      values:
+        - key: "{{enumCommands}}"
+          enabledValues:
+            - 'play'
+            - 'pause'
+            - 'stop'
+        - key: "playbackStatus.value"
+          enabledValues:
```

Comment:
There are issues with the "enabledValues" config and it does not currently work in prod. The ticket to fix this is currently in progress and can be tracked here [CODA-2736](https://smartthings.atlassian.net/browse/CODA-2736)

This change should probably wait until the CODA dependency is complete before merging.

Reply:
> 

@ctowns : Thank you for sharing.